server/apps/io/reuters.py

```diff
@@ -102,9 +102,12 @@ def get_ids(self, channel, last_updated, updated):
         ids = []
         payload = {'channel': channel, 'fieldsRef': 'id'}
         payload['dateRange'] = "%s-%s" % (self.format_date(last_updated), self.format_date(updated))
+        logger.info('Reuters requesting Date Range |{}| for channel {}'.format(payload['dateRange'], channel))
         tree = self.get_tree('items', payload)
         for result in tree.findall('result'):
-            ids.append(result.find('guid').text)
+            id = result.find('guid').text
```

Comment:
I suggest picking a different variable name to avoid any confusion with the built-in `id()` function.